internal/export/client/init.go

```diff
@@ -55,8 +58,8 @@ func Init(conf ConfigurationStruct, l *zap.Logger) error {
 	var err error
 
 	// Create a database client
-	dbc, err = clients.NewDBClient(clients.DBConfiguration{
-		DbType:       clients.GetDatabaseType(conf.DBType),
+	mongo.NewClient(db.Configuration{
```

Comment:
@brandonforster As you can see here, the `dbc, err` on the left side of the assignment was removed. This is most likely the cause of the panic.